src/components/TodoList/TodoList.spec.tsx

```diff
@@ -48,7 +49,7 @@ describe('TodoList', () => {
       },
     ];
 
-    mount(<TodoList todos={todos} />);
+    mount(<TodoList todos={todos} users={[]} />);
```

Comment:
The 'users' prop is passed as an empty array to the TodoList component. According to the checklist, you should prepare the data in one place (e.g., App component) and pass it to child components. If the users' data is required by the TodoList for rendering or functionality, it should be included in the test setup to accurately reflect the component's usage.